### PR TITLE
Add docs for `DROP CACHE <query text>`

### PR DESCRIPTION
**Pull request by ethan-readyset**
Associated with https://github.com/readysettech/readyset/pull/928

### CODE REVIEW

**Review by zoready — APPROVED** [review]
LGTM

**Comment by ethan-readyset** [comment]
Thank ya! I'm gonna hold off on merging this until the linked PR is merged

**Review by aglover — APPROVED** [review]
LGTM

**Pull request merged**
